AlfrescoApp/App Delegate/AppDelegate.m

```diff
@@ -108,11 +108,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
             [hockeyManager.authenticator authenticateInstallation];
         }
         
-        // Flurry Analytics
-        if (FLURRY_API_KEY.length > 0)
-        {
-            [[AnalyticsManager sharedManager] startAnalytics];
-        }
```

Comment:
Just seen Flurry code has moved to lower down. I'll raise the question with PM as to it's continued usage.